WEBAPP.md

```diff
@@ -1,23 +1,21 @@
-# 開源星手村 - 網頁版
-
 # Open Star Ter Village - Web App
```

Comment:
Should we standardize the usage to 'Open Star Ter Village' or 'Open StarTer Village'?

Reply:
Let's align it to be Open Star Ter Village as same as our project name.